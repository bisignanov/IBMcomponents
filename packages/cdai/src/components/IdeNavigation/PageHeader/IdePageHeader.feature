Feature: IDE Page Header component
  As an ICP4I product
  I want to be able to embed a common page header component.

Background:
    Given Setup of the IdePageHeader world

Scenario Outline: When I provide a set properties, these are rendered by the IdePageHeader component
    Given a set of properties called "<props>" provided to the component
    When the component is rendered
    Then the content of "<props>" is rendered as expected

    Examples:
    | props                                 |
    | none                                  |
    | breadcrumb                            |
    | tabs                                  |
    | breadcrumb,heading                    |
    | breadcrumb,tabs                       |
    | breadcrumb,others                     |
    | breadcrumb,custom                     |
    | breadcrumb,heading,tabs               |
    | breadcrumb,heading,others             |
    | breadcrumb,heading,custom             |
    | breadcrumb,heading,tabs,others        |
    | breadcrumb,heading,tabs,custom        |
    | breadcrumb,heading,tabs,others,custom |
    | header,headerRenderFunction           |
    | headerRenderFunction                  |
  