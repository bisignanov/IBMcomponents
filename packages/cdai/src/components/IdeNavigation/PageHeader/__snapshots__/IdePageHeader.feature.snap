// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feature: IDE Page Header component When I provide a set properties, these are rendered by the IdePageHeader component 1`] = `
Array [
  <section
    className="ide-page-header "
    data-ui-id="IdePageHeader"
  >
    <IdePageContent
      bleedClassName="ide-page-header__backgroundColour"
      className=""
    >
      <div
        className="bx--row"
      >
        <div
          className="bx--col"
        >
          <div
            className="ide-page-header--breadcrumb-container"
            data-ui-id="IdePageHeader--breadcrumb-container"
          />
          <div
            className="ide-page-header--heading-container"
            data-ui-id="IdePageHeader--heading-container"
          />
          <div
            className="ide-page-header--subtitle-container"
            data-ui-id="IdePageHeader--subtitle-container"
          />
          <div
            className="ide-page-header--tabs-container__empty"
            data-ui-id="IdePageHeader--tabs-container"
          />
        </div>
      </div>
    </IdePageContent>
  </section>,
]
`;

exports[`Feature: IDE Page Header component When I provide a set properties, these are rendered by the IdePageHeader component 2`] = `
Array [
  <section
    className="ide-page-header "
    data-ui-id="IdePageHeader"
  >
    <IdePageContent
      bleedClassName="ide-page-header__backgroundColour"
      className=""
    >
      <div
        className="bx--row"
      >
        <div
          className="bx--col"
        >
          <div
            className="ide-page-header--breadcrumb-container"
            data-ui-id="IdePageHeader--breadcrumb-container"
          >
            <Breadcrumb
              data-ui-id="breadcrumb"
              noTrailingSlash={true}
            >
              <BreadcrumbItem>
                <a
                  href="#top"
                >
                  items
                </a>
              </BreadcrumbItem>
              <BreadcrumbItem
                isCurrentPage={true}
              >
                <div>
                  item
                </div>
              </BreadcrumbItem>
            </Breadcrumb>
          </div>
        </div>
      </div>
    </IdePageContent>
  </section>,
]
`;

exports[`Feature: IDE Page Header component When I provide a set properties, these are rendered by the IdePageHeader component 3`] = `
Array [
  <section
    className="ide-page-header "
    data-ui-id="IdePageHeader"
  >
    <IdePageContent
      bleedClassName="ide-page-header__backgroundColour"
      className=""
    >
      <div
        className="bx--row"
      >
        <div
          className="bx--col"
        >
          <div
            className="ide-page-header--tabs-container"
            data-ui-id="IdePageHeader--tabs-container"
          >
            <Tabs
              role="navigation"
              selected={0}
              selectionMode="automatic"
              type="default"
            >
              <Tab
                data-ui-id="tab1"
                href="#"
                label="Tab one"
                onClick={[Function]}
                onKeyDown={[Function]}
                renderContent={[Function]}
                role="presentation"
                selected={false}
                tabIndex={0}
              />
              <Tab
                data-ui-id="tab2"
                href="#"
                label="Tab two"
                onClick={[Function]}
                onKeyDown={[Function]}
                renderContent={[Function]}
                role="presentation"
                selected={false}
                tabIndex={0}
              />
            </Tabs>
          </div>
        </div>
      </div>
    </IdePageContent>
  </section>,
]
`;

exports[`Feature: IDE Page Header component When I provide a set properties, these are rendered by the IdePageHeader component 4`] = `
Array [
  <section
    className="ide-page-header "
    data-ui-id="IdePageHeader"
  >
    <IdePageContent
      bleedClassName="ide-page-header__backgroundColour"
      className=""
    >
      <div
        className="bx--row"
      >
        <div
          className="bx--col"
        >
          <div
            className="ide-page-header--breadcrumb-container"
            data-ui-id="IdePageHeader--breadcrumb-container"
          >
            <Breadcrumb
              data-ui-id="breadcrumb"
              noTrailingSlash={true}
            >
              <BreadcrumbItem>
                <a
                  href="#top"
                >
                  items
                </a>
              </BreadcrumbItem>
              <BreadcrumbItem
                isCurrentPage={true}
              >
                <div>
                  item
                </div>
              </BreadcrumbItem>
            </Breadcrumb>
          </div>
          <div
            className="ide-page-header--heading-container"
            data-ui-id="IdePageHeader--heading-container"
          >
            <p
              className="ide-page-header--heading"
              data-ui-id="IdePageHeader--heading"
            >
              my heading
            </p>
          </div>
          <div
            className="ide-page-header--subtitle-container"
            data-ui-id="IdePageHeader--subtitle-container"
          />
          <div
            className="ide-page-header--tabs-container__empty"
            data-ui-id="IdePageHeader--tabs-container"
          />
        </div>
      </div>
    </IdePageContent>
  </section>,
]
`;

exports[`Feature: IDE Page Header component When I provide a set properties, these are rendered by the IdePageHeader component 5`] = `
Array [
  <section
    className="ide-page-header "
    data-ui-id="IdePageHeader"
  >
    <IdePageContent
      bleedClassName="ide-page-header__backgroundColour"
      className=""
    >
      <div
        className="bx--row"
      >
        <div
          className="bx--col"
        >
          <div
            className="ide-page-header--breadcrumb-container"
            data-ui-id="IdePageHeader--breadcrumb-container"
          >
            <Breadcrumb
              data-ui-id="breadcrumb"
              noTrailingSlash={true}
            >
              <BreadcrumbItem>
                <a
                  href="#top"
                >
                  items
                </a>
              </BreadcrumbItem>
              <BreadcrumbItem
                isCurrentPage={true}
              >
                <div>
                  item
                </div>
              </BreadcrumbItem>
            </Breadcrumb>
          </div>
          <div
            className="ide-page-header--heading-container"
            data-ui-id="IdePageHeader--heading-container"
          />
          <div
            className="ide-page-header--subtitle-container"
            data-ui-id="IdePageHeader--subtitle-container"
          />
          <div
            className="ide-page-header--tabs-container"
            data-ui-id="IdePageHeader--tabs-container"
          >
            <Tabs
              role="navigation"
              selected={0}
              selectionMode="automatic"
              type="default"
            >
              <Tab
                data-ui-id="tab1"
                href="#"
                label="Tab one"
                onClick={[Function]}
                onKeyDown={[Function]}
                renderContent={[Function]}
                role="presentation"
                selected={false}
                tabIndex={0}
              />
              <Tab
                data-ui-id="tab2"
                href="#"
                label="Tab two"
                onClick={[Function]}
                onKeyDown={[Function]}
                renderContent={[Function]}
                role="presentation"
                selected={false}
                tabIndex={0}
              />
            </Tabs>
          </div>
        </div>
      </div>
    </IdePageContent>
  </section>,
]
`;

exports[`Feature: IDE Page Header component When I provide a set properties, these are rendered by the IdePageHeader component 6`] = `
Array [
  <section
    className="ide-page-header storybook--use-width storybook--ide-page-header-heading"
    data-ui-id="IdePageHeader"
  >
    <IdePageContent
      bleedClassName="ide-page-header__backgroundColour"
      className=""
    >
      <div
        className="bx--row"
      >
        <div
          className="bx--col"
        >
          <div
            className="ide-page-header--breadcrumb-container"
            data-ui-id="IdePageHeader--breadcrumb-container"
          >
            <Breadcrumb
              data-ui-id="breadcrumb"
              noTrailingSlash={true}
            >
              <BreadcrumbItem>
                <a
                  href="#top"
                >
                  items
                </a>
              </BreadcrumbItem>
              <BreadcrumbItem
                isCurrentPage={true}
              >
                <div>
                  item
                </div>
              </BreadcrumbItem>
            </Breadcrumb>
          </div>
        </div>
      </div>
    </IdePageContent>
  </section>,
]
`;

exports[`Feature: IDE Page Header component When I provide a set properties, these are rendered by the IdePageHeader component 7`] = `
Array [
  <section
    className="ide-page-header "
    data-ui-id="IdePageHeader"
  >
    <IdePageContent
      bleedClassName="ide-page-header__backgroundColour"
      className=""
    >
      <div
        className="bx--row"
      >
        <div
          className="bx--col"
        >
          <div
            className="ide-page-header--breadcrumb-container"
            data-ui-id="IdePageHeader--breadcrumb-container"
          >
            <Breadcrumb
              data-ui-id="breadcrumb"
              noTrailingSlash={true}
            >
              <BreadcrumbItem>
                <a
                  href="#top"
                >
                  items
                </a>
              </BreadcrumbItem>
              <BreadcrumbItem
                isCurrentPage={true}
              >
                <div>
                  item
                </div>
              </BreadcrumbItem>
            </Breadcrumb>
          </div>
        </div>
        <div
          className="bx--col ide-page-header--custom-content-container"
          data-ui-id="IdePageHeader--custom-content-container"
        >
          <div
            className="storybook--idepageheader-button"
            data-ui-id="custom"
          >
            <ForwardRef(Button)
              disabled={false}
              kind="primary"
              size="default"
              tabIndex={0}
              tooltipAlignment="center"
              tooltipPosition="top"
              type="button"
            >
              Add new topic
            </ForwardRef(Button)>
          </div>
        </div>
      </div>
    </IdePageContent>
  </section>,
]
`;

exports[`Feature: IDE Page Header component When I provide a set properties, these are rendered by the IdePageHeader component 8`] = `
Array [
  <section
    className="ide-page-header "
    data-ui-id="IdePageHeader"
  >
    <IdePageContent
      bleedClassName="ide-page-header__backgroundColour"
      className=""
    >
      <div
        className="bx--row"
      >
        <div
          className="bx--col"
        >
          <div
            className="ide-page-header--breadcrumb-container"
            data-ui-id="IdePageHeader--breadcrumb-container"
          >
            <Breadcrumb
              data-ui-id="breadcrumb"
              noTrailingSlash={true}
            >
              <BreadcrumbItem>
                <a
                  href="#top"
                >
                  items
                </a>
              </BreadcrumbItem>
              <BreadcrumbItem
                isCurrentPage={true}
              >
                <div>
                  item
                </div>
              </BreadcrumbItem>
            </Breadcrumb>
          </div>
          <div
            className="ide-page-header--heading-container"
            data-ui-id="IdePageHeader--heading-container"
          >
            <p
              className="ide-page-header--heading"
              data-ui-id="IdePageHeader--heading"
            >
              my heading
            </p>
          </div>
          <div
            className="ide-page-header--subtitle-container"
            data-ui-id="IdePageHeader--subtitle-container"
          />
          <div
            className="ide-page-header--tabs-container"
            data-ui-id="IdePageHeader--tabs-container"
          >
            <Tabs
              role="navigation"
              selected={0}
              selectionMode="automatic"
              type="default"
            >
              <Tab
                data-ui-id="tab1"
                href="#"
                label="Tab one"
                onClick={[Function]}
                onKeyDown={[Function]}
                renderContent={[Function]}
                role="presentation"
                selected={false}
                tabIndex={0}
              />
              <Tab
                data-ui-id="tab2"
                href="#"
                label="Tab two"
                onClick={[Function]}
                onKeyDown={[Function]}
                renderContent={[Function]}
                role="presentation"
                selected={false}
                tabIndex={0}
              />
            </Tabs>
          </div>
        </div>
      </div>
    </IdePageContent>
  </section>,
]
`;

exports[`Feature: IDE Page Header component When I provide a set properties, these are rendered by the IdePageHeader component 9`] = `
Array [
  <section
    className="ide-page-header storybook--use-width storybook--ide-page-header-heading"
    data-ui-id="IdePageHeader"
  >
    <IdePageContent
      bleedClassName="ide-page-header__backgroundColour"
      className=""
    >
      <div
        className="bx--row"
      >
        <div
          className="bx--col"
        >
          <div
            className="ide-page-header--breadcrumb-container"
            data-ui-id="IdePageHeader--breadcrumb-container"
          >
            <Breadcrumb
              data-ui-id="breadcrumb"
              noTrailingSlash={true}
            >
              <BreadcrumbItem>
                <a
                  href="#top"
                >
                  items
                </a>
              </BreadcrumbItem>
              <BreadcrumbItem
                isCurrentPage={true}
              >
                <div>
                  item
                </div>
              </BreadcrumbItem>
            </Breadcrumb>
          </div>
          <div
            className="ide-page-header--heading-container"
            data-ui-id="IdePageHeader--heading-container"
          >
            <p
              className="ide-page-header--heading"
              data-ui-id="IdePageHeader--heading"
            >
              my heading
            </p>
          </div>
          <div
            className="ide-page-header--subtitle-container"
            data-ui-id="IdePageHeader--subtitle-container"
          />
          <div
            className="ide-page-header--tabs-container__empty"
            data-ui-id="IdePageHeader--tabs-container"
          />
        </div>
      </div>
    </IdePageContent>
  </section>,
]
`;

exports[`Feature: IDE Page Header component When I provide a set properties, these are rendered by the IdePageHeader component 10`] = `
Array [
  <section
    className="ide-page-header "
    data-ui-id="IdePageHeader"
  >
    <IdePageContent
      bleedClassName="ide-page-header__backgroundColour"
      className=""
    >
      <div
        className="bx--row"
      >
        <div
          className="bx--col"
        >
          <div
            className="ide-page-header--breadcrumb-container"
            data-ui-id="IdePageHeader--breadcrumb-container"
          >
            <Breadcrumb
              data-ui-id="breadcrumb"
              noTrailingSlash={true}
            >
              <BreadcrumbItem>
                <a
                  href="#top"
                >
                  items
                </a>
              </BreadcrumbItem>
              <BreadcrumbItem
                isCurrentPage={true}
              >
                <div>
                  item
                </div>
              </BreadcrumbItem>
            </Breadcrumb>
          </div>
          <div
            className="ide-page-header--heading-container"
            data-ui-id="IdePageHeader--heading-container"
          >
            <p
              className="ide-page-header--heading"
              data-ui-id="IdePageHeader--heading"
            >
              my heading
            </p>
          </div>
          <div
            className="ide-page-header--subtitle-container"
            data-ui-id="IdePageHeader--subtitle-container"
          />
          <div
            className="ide-page-header--tabs-container__empty"
            data-ui-id="IdePageHeader--tabs-container"
          />
        </div>
        <div
          className="bx--col ide-page-header--custom-content-container"
          data-ui-id="IdePageHeader--custom-content-container"
        >
          <div
            className="storybook--idepageheader-button"
            data-ui-id="custom"
          >
            <ForwardRef(Button)
              disabled={false}
              kind="primary"
              size="default"
              tabIndex={0}
              tooltipAlignment="center"
              tooltipPosition="top"
              type="button"
            >
              Add new topic
            </ForwardRef(Button)>
          </div>
        </div>
      </div>
    </IdePageContent>
  </section>,
]
`;

exports[`Feature: IDE Page Header component When I provide a set properties, these are rendered by the IdePageHeader component 11`] = `
Array [
  <section
    className="ide-page-header storybook--use-width storybook--ide-page-header-heading"
    data-ui-id="IdePageHeader"
  >
    <IdePageContent
      bleedClassName="ide-page-header__backgroundColour"
      className=""
    >
      <div
        className="bx--row"
      >
        <div
          className="bx--col"
        >
          <div
            className="ide-page-header--breadcrumb-container"
            data-ui-id="IdePageHeader--breadcrumb-container"
          >
            <Breadcrumb
              data-ui-id="breadcrumb"
              noTrailingSlash={true}
            >
              <BreadcrumbItem>
                <a
                  href="#top"
                >
                  items
                </a>
              </BreadcrumbItem>
              <BreadcrumbItem
                isCurrentPage={true}
              >
                <div>
                  item
                </div>
              </BreadcrumbItem>
            </Breadcrumb>
          </div>
          <div
            className="ide-page-header--heading-container"
            data-ui-id="IdePageHeader--heading-container"
          >
            <p
              className="ide-page-header--heading"
              data-ui-id="IdePageHeader--heading"
            >
              my heading
            </p>
          </div>
          <div
            className="ide-page-header--subtitle-container"
            data-ui-id="IdePageHeader--subtitle-container"
          />
          <div
            className="ide-page-header--tabs-container"
            data-ui-id="IdePageHeader--tabs-container"
          >
            <Tabs
              role="navigation"
              selected={0}
              selectionMode="automatic"
              type="default"
            >
              <Tab
                data-ui-id="tab1"
                href="#"
                label="Tab one"
                onClick={[Function]}
                onKeyDown={[Function]}
                renderContent={[Function]}
                role="presentation"
                selected={false}
                tabIndex={0}
              />
              <Tab
                data-ui-id="tab2"
                href="#"
                label="Tab two"
                onClick={[Function]}
                onKeyDown={[Function]}
                renderContent={[Function]}
                role="presentation"
                selected={false}
                tabIndex={0}
              />
            </Tabs>
          </div>
        </div>
      </div>
    </IdePageContent>
  </section>,
]
`;

exports[`Feature: IDE Page Header component When I provide a set properties, these are rendered by the IdePageHeader component 12`] = `
Array [
  <section
    className="ide-page-header "
    data-ui-id="IdePageHeader"
  >
    <IdePageContent
      bleedClassName="ide-page-header__backgroundColour"
      className=""
    >
      <div
        className="bx--row"
      >
        <div
          className="bx--col"
        >
          <div
            className="ide-page-header--breadcrumb-container"
            data-ui-id="IdePageHeader--breadcrumb-container"
          >
            <Breadcrumb
              data-ui-id="breadcrumb"
              noTrailingSlash={true}
            >
              <BreadcrumbItem>
                <a
                  href="#top"
                >
                  items
                </a>
              </BreadcrumbItem>
              <BreadcrumbItem
                isCurrentPage={true}
              >
                <div>
                  item
                </div>
              </BreadcrumbItem>
            </Breadcrumb>
          </div>
          <div
            className="ide-page-header--heading-container"
            data-ui-id="IdePageHeader--heading-container"
          >
            <p
              className="ide-page-header--heading"
              data-ui-id="IdePageHeader--heading"
            >
              my heading
            </p>
          </div>
          <div
            className="ide-page-header--subtitle-container"
            data-ui-id="IdePageHeader--subtitle-container"
          />
          <div
            className="ide-page-header--tabs-container"
            data-ui-id="IdePageHeader--tabs-container"
          >
            <Tabs
              role="navigation"
              selected={0}
              selectionMode="automatic"
              type="default"
            >
              <Tab
                data-ui-id="tab1"
                href="#"
                label="Tab one"
                onClick={[Function]}
                onKeyDown={[Function]}
                renderContent={[Function]}
                role="presentation"
                selected={false}
                tabIndex={0}
              />
              <Tab
                data-ui-id="tab2"
                href="#"
                label="Tab two"
                onClick={[Function]}
                onKeyDown={[Function]}
                renderContent={[Function]}
                role="presentation"
                selected={false}
                tabIndex={0}
              />
            </Tabs>
          </div>
        </div>
        <div
          className="bx--col ide-page-header--custom-content-container"
          data-ui-id="IdePageHeader--custom-content-container"
        >
          <div
            className="storybook--idepageheader-button"
            data-ui-id="custom"
          >
            <ForwardRef(Button)
              disabled={false}
              kind="primary"
              size="default"
              tabIndex={0}
              tooltipAlignment="center"
              tooltipPosition="top"
              type="button"
            >
              Add new topic
            </ForwardRef(Button)>
          </div>
        </div>
      </div>
    </IdePageContent>
  </section>,
]
`;

exports[`Feature: IDE Page Header component When I provide a set properties, these are rendered by the IdePageHeader component 13`] = `
Array [
  <section
    className="ide-page-header storybook--use-width storybook--ide-page-header-heading"
    data-ui-id="IdePageHeader"
  >
    <IdePageContent
      bleedClassName="ide-page-header__backgroundColour"
      className=""
    >
      <div
        className="bx--row"
      >
        <div
          className="bx--col"
        >
          <div
            className="ide-page-header--breadcrumb-container"
            data-ui-id="IdePageHeader--breadcrumb-container"
          >
            <Breadcrumb
              data-ui-id="breadcrumb"
              noTrailingSlash={true}
            >
              <BreadcrumbItem>
                <a
                  href="#top"
                >
                  items
                </a>
              </BreadcrumbItem>
              <BreadcrumbItem
                isCurrentPage={true}
              >
                <div>
                  item
                </div>
              </BreadcrumbItem>
            </Breadcrumb>
          </div>
          <div
            className="ide-page-header--heading-container"
            data-ui-id="IdePageHeader--heading-container"
          >
            <p
              className="ide-page-header--heading"
              data-ui-id="IdePageHeader--heading"
            >
              my heading
            </p>
          </div>
          <div
            className="ide-page-header--subtitle-container"
            data-ui-id="IdePageHeader--subtitle-container"
          />
          <div
            className="ide-page-header--tabs-container"
            data-ui-id="IdePageHeader--tabs-container"
          >
            <Tabs
              role="navigation"
              selected={0}
              selectionMode="automatic"
              type="default"
            >
              <Tab
                data-ui-id="tab1"
                href="#"
                label="Tab one"
                onClick={[Function]}
                onKeyDown={[Function]}
                renderContent={[Function]}
                role="presentation"
                selected={false}
                tabIndex={0}
              />
              <Tab
                data-ui-id="tab2"
                href="#"
                label="Tab two"
                onClick={[Function]}
                onKeyDown={[Function]}
                renderContent={[Function]}
                role="presentation"
                selected={false}
                tabIndex={0}
              />
            </Tabs>
          </div>
        </div>
        <div
          className="bx--col ide-page-header--custom-content-container"
          data-ui-id="IdePageHeader--custom-content-container"
        >
          <div
            className="storybook--idepageheader-button"
            data-ui-id="custom"
          >
            <ForwardRef(Button)
              disabled={false}
              kind="primary"
              size="default"
              tabIndex={0}
              tooltipAlignment="center"
              tooltipPosition="top"
              type="button"
            >
              Add new topic
            </ForwardRef(Button)>
          </div>
        </div>
      </div>
    </IdePageContent>
  </section>,
]
`;

exports[`Feature: IDE Page Header component When I provide a set properties, these are rendered by the IdePageHeader component 14`] = `
Array [
  <section
    className="ide-page-header "
    data-ui-id="IdePageHeader"
  >
    <IdePageContent
      bleedClassName="ide-page-header__backgroundColour"
      className=""
    >
      <div
        className="bx--row"
      >
        <div
          className="bx--col"
        >
          <div
            className="ide-page-header--breadcrumb-container"
            data-ui-id="IdePageHeader--breadcrumb-container"
          />
          <div
            className="ide-page-header--heading-container"
            data-ui-id="IdePageHeader--heading-container"
          >
            <React.Fragment>
              <ForwardRef(Bee32)
                data-ui-id="custom-header-content"
              />
            </React.Fragment>
          </div>
          <div
            className="ide-page-header--subtitle-container"
            data-ui-id="IdePageHeader--subtitle-container"
          />
          <div
            className="ide-page-header--tabs-container__empty"
            data-ui-id="IdePageHeader--tabs-container"
          />
        </div>
      </div>
    </IdePageContent>
  </section>,
]
`;

exports[`Feature: IDE Page Header component When I provide a set properties, these are rendered by the IdePageHeader component 15`] = `
Array [
  <section
    className="ide-page-header "
    data-ui-id="IdePageHeader"
  >
    <IdePageContent
      bleedClassName="ide-page-header__backgroundColour"
      className=""
    >
      <div
        className="bx--row"
      >
        <div
          className="bx--col"
        >
          <div
            className="ide-page-header--breadcrumb-container"
            data-ui-id="IdePageHeader--breadcrumb-container"
          />
          <div
            className="ide-page-header--heading-container"
            data-ui-id="IdePageHeader--heading-container"
          >
            <React.Fragment>
              <ForwardRef(Bee32)
                data-ui-id="custom-header-content"
              />
            </React.Fragment>
          </div>
          <div
            className="ide-page-header--subtitle-container"
            data-ui-id="IdePageHeader--subtitle-container"
          />
          <div
            className="ide-page-header--tabs-container__empty"
            data-ui-id="IdePageHeader--tabs-container"
          />
        </div>
      </div>
    </IdePageContent>
  </section>,
]
`;
